tasks/DisableCpuFreqScaling.rb

```diff
@@ -0,0 +1,16 @@
+class DisableCpuFreqScaling
+  include Genesis::Framework::Task
+
+  init do
+    log "Make sure acpi-cpufreq is enabled" 
+    run_cmd('modprobe acpi-cpufreq')
```

Comment:
Can this be accomplished instead by starting the corresponding cpufreq service? Less tight coupling to the os implementation?